app/graph/mutations/generate_two_factor_backup_codes_mutation.rb

```diff
@@ -0,0 +1,18 @@
+GenerateTwoFactorBackupCodesMutation = GraphQL::Relay::Mutation.define do
+  name 'GenerateTwoFactorBackupCodes'
+
+  input_field :id, !types.Int
+  
+  return_field :success, types.Boolean
+  return_field :codes, JsonStringType
+
+  resolve -> (_root, inputs, _ctx) {
+    user = User.where(id: inputs[:id]).last
```

Comment:
Please be sure that this user is the same as the current_api_user - we don't one user to request the 2FA code for another user :)

Reply:
Fixed